apps/sms/views/conversation/js/recipients.js

```diff
@@ -431,7 +431,7 @@
     new GestureDetector(outer).startDetecting();
 
     // Set focus on the last "placeholder" element
-    this.reset().focus();
+    this.reset();
```

Comment:
Mmmm, can't really find this case when it's maybe needed... So I'm fine if you remove it. And looks like ```// Set focus on the last "placeholder" element``` comment is no longer accurate.